core/src/por/api.ts

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios'
+import { OraklError, OraklErrorCode } from '../errors'
+import { CHAIN, ORAKL_NETWORK_API_URL } from '../settings'
+import { IAggregator } from '../types'
+import { buildUrl } from '../utils'
+import { IData } from './types'
+
+export async function loadAggregator({ aggregatorHash }: { aggregatorHash: string }) {
+  const chain = CHAIN
+  try {
+    const url = buildUrl(ORAKL_NETWORK_API_URL, `aggregator/${aggregatorHash}/${chain}`)
```

Comment:
We could pass the `CHAIN` variable directly without creating a transitory variable.

```suggestion
    const url = buildUrl(ORAKL_NETWORK_API_URL, `aggregator/${aggregatorHash}/${CHAIN}`)
```